### ISSUE TITLE
Module does not export directives

### ISSUE THREAD

**Issue by OrangeUtan** [henriquehbr/svelte-typewriter#80]
Dependencies:
- "svelte": "^3.49.0"
- "svelte-typewriter": "3.0.0-beta.0"

```js
<script>
	import { concurrent } from 'svelte-typewriter';
	import Typewriter from 'svelte-typewriter';
	import { onMount } from 'svelte';

	let x = 1;

  onMount(() => {
    setInterval(() => x++, 1000);
  });
</script>

<p use:concurrent={{ interval: 20 }}>{x}</p>

<Typewriter>
  <p>{x}</p>
</Typewriter>

```
[REPL](https://svelte.dev/repl/a62e0afcdcb44cad982567f952e98449?version=3.49.0)

Importing and using the `Typewriter` works fine, but importing directives does not work correctly.
TypesScript complains: `Module '"svelte-typewriter"' has no exported member 'concurrent'. Did you mean to use 'import concurrent from "svelte-typewriter"' instead?ts(2614)`
Interestingly the animation still plays, but only once.

**Comment by henriquehbr** [comment]
Hey @OrangeUtan, thanks for reporting this problem, this happens because the types of the library are outdated, i'll be updating them to prevent problems like this, and i'll let you know once this problem is fixed

And as you stated, as it's only a Typescript error, it shouldn't affect the runtime, so it's expected that your application work just fine, the animation playing only once it's also expected, because you're using the "concurrent" mode, there are plenty of [other modes](https://github.com/henriquehbr/svelte-typewriter#modes) that you can use, including a "loop" one if you don't want the animation to play only once

**Comment by henriquehbr** [comment]
I've added type definitions for the directives on version [3.0.0-beta.1](https://git.sr.ht/~henriquehbr/svelte-typewriter/tree/main/item/CHANGELOG.md#300-beta1---2022-07-11), please, let me know if this fixes the problem for you, and feel free to close this issue if that's the case

**Comment by OrangeUtan** [comment]
Awesome, thanks for the quick response. I just installed `svelte-typewriter@^3.0.0-beta.1`  and I think it might be fixed, but only if I missuderstood the usage. Instead of no directives at all, now at least the types are exported.

The example in the README still doesn't work though, it complains that `concurrent` is a type:
```ts
<script lang="ts">
    import { concurrent } from 'svelte-typewriter';
</script>

<p use:concurrent={{ interval: 30 }}>Testing the typewriter effect</p>
```

Importing the directive directly from the component works:
```ts
<script lang="ts">
    import { concurrent } from 'svelte-typewriter/src/Typewriter.svelte';
</script>

<p use:concurrent={{ interval: 30 }}>Testing the typewriter effect</p>
```

**Comment by henriquehbr** [comment]
My apologies for the silly mistake, this now should be fixed on version [3.0.1](https://git.sr.ht/~henriquehbr/svelte-typewriter/refs/v3.0.1)

**Comment by henriquehbr** [comment]
As the project is migrating to sourcehut, i'll be closing this issue, if you feel that the reported problem haven't been solved, feel free to reopen it [here](https://todo.sr.ht/~henriquehbr/svelte-typewriter), otherwise, you can just ignore this message
